src/deploy/executor.rs

```diff
@@ -73,7 +59,7 @@ where
 
 		for hook in &profile.pre_hooks {
 			log::info!("Executing pre hook: `{:?}`", hook);
-			hook.execute().unwrap();
+			hook.execute()?;
```

Comment:
Shouldn't be `wrap_err("Failed to execute pre-hook")?` here too?

Reply:
Done